docker/dockerfiles/be/be-ubuntu.Dockerfile

```diff
@@ -32,6 +32,10 @@ RUN apt-get update -y && apt-get install -y --no-install-recommends \
         ln -fs /usr/share/zoneinfo/UTC /etc/localtime && \
         dpkg-reconfigure -f noninteractive tzdata && \
         rm -rf /var/lib/apt/lists/*
+
+# Install locales and generate en_US.UTF-8, which is required by a downstream scalar udf STRING_TO_DOUBLE.
+RUN apt-get install -y locales && locale-gen en_US.UTF-8
```

Comment:
will this as `rm -rf /var/lib/apt/lists/*` happened before?
suggest combine it to the first RUN, following docker best practice:
https://docs.docker.com/develop/develop-images/instructions/#apt-get 

```
RUN apt-get update -y && apt-get install -y --no-install-recommends \
        binutils-dev default-jdk python2 mysql-client curl vim tree net-tools less tzdata linux-tools-common linux-tools-generic locales && \
        ln -fs /usr/share/zoneinfo/UTC /etc/localtime && \
        dpkg-reconfigure -f noninteractive tzdata && \
       locale-gen en_US.UTF-8 && \
        rm -rf /var/lib/apt/lists/*
```

Reply:
Changed. PTAL.